docs/docs/queries/structure.md

```diff
@@ -1,44 +1,207 @@
+4. **CALENDAR**: A calendar view displaying each hit via a dot on its referred date.
 
-The most basic example of a dataview query is:
+The Query Type is the **only mandatory command in a query**. Everything else is optional.
 
 ~~~
+Lists all pages in your vault as a table with one column containing the page link
+```dataview
+TABLE
+```
+
+Lists all pages in your vault as a bullet point list
 ```dataview
 LIST
 ```
+
+Lists all tasks (completed or not) in your vault
+```dataview
+TASK
+```
+
+Renders a Calendar view where each page is represented as a dot on its creation date.
+```dataview
+CALENDAR file.cday
+```
 ~~~
 
-which will list **all files in your vault**. Find more examples [here](../resources/examples.md).
\ No newline at end of file
+!!! info "Read more about the available types and how to use them [here](./query-types.md)."
+
+## Choose your source
+
+Additionally to the Query Types, you have several **Data Commands** available that help you restrict, refine, sort or group your query. One of these query commands is the **FROM** statement, which is a bit special.
+
+!!! info "`FROM` determines which set of notes should be included in your query depending on given [sources](../../reference/sources)."
+
+You can add **zero or one** `FROM` data command to your query, right after your Query Type. You cannot add multiple FROM statements and you cannot add it after other Data Commands.
+
+`FROM` takes [sources](../../reference/sources) as argument and **restricts the pages your query will collect to the given sources**.
+
+~~~
+Lists all pages inside the folder Books and its sub folders
+```dataview
+LIST
+FROM "Books"
+```
+
+Lists all pages that include the tag #status/open or #status/wip
+```dataview
+LIST
+FROM #status/open OR #status/wip
+```
+
+Lists all pages that have either the tag #assignment and are inside folder "30 School" (or its sub folders), or are inside folder "30 School/32 Homeworks" and are linked on the page School Dashboard Current To Dos
+```dataview
+LIST
+FROM (#assignment AND "30 School") OR ("30 School/32 Homeworks" AND outgoing([[School Dashboard Current To Dos]]))
+```
+
+~~~
+
+!!! info "Read more about `FROM` [here](./data-commands.md#from)."
+
+## Filter, sort, group or limit results
+
+Additionally to the Query Types and the **Data command** `FROM` that's explained above, you have several other **Data Commands** available that help you restrict, refine, sort or group your query. 
+
+All data commands except the `FROM` command can be used **multiple times in any order** (as long as they come after the Query Type and `FROM`, if `FROM` is used at all). They'll be excuted in the order they are written.
+
+Available are:
+
+1. **FROM** like explained [above](#choose-your-source).
+2. **WHERE**: Filter notes based on information **inside** notes, the meta data fields.
+3. **SORT**: Sorts your results depending on a field and a direction.
+4. **GROUP BY**: Bundles up several results into one result row.
```

Comment:
Bundles up results into groups (potential for misinterpreting as "bundled into a single row")